lib/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Util = require('util');
+
+const Config = require('../config');
+const Movies = require('./server');
+
+Movies.start(() => {
+    Util.log(`Server started on port: ${Config.PORT}`);
```

Comment:
Here, and in other places further down, you are indenting 4 spaces instead of the standard 2 spaces. The linter should be catching that. Is it not?